deployment/streaming/scripts/define-staging-view-refresher.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+if [ -z ${VERSION_TAG+x} ]; then
+    echo "Do not run this script directly.  Use the Makefile in the parent directory."
+    exit 1
+fi
+
+aws ecs register-task-definition \
+    --family osmesa-stats-view-refresher-staging \
+    --task-role-arn "arn:aws:iam::${IAM_ACCOUNT}:role/ECSTaskS3" \
+    --execution-role-arn "arn:aws:iam::${IAM_ACCOUNT}:role/ecsTaskExecutionRole" \
+    --network-mode awsvpc \
+    --requires-compatibilities EC2 FARGATE \
+    --cpu "0.25 vCPU" \
+    --memory "0.5 GB" \
+    --container-definitions "[
+            {
+              \"logConfiguration\": {
+                \"logDriver\": \"awslogs\",
+                \"options\": {
+	          \"awslogs-group\": \"/ecs/${AWS_LOG_GROUP}-staging\",
+	          \"awslogs-region\": \"${AWS_REGION}\",
+                  \"awslogs-stream-prefix\": \"ecs\"
+                }
+              },
+              \"command\": [
+                \"refresh-views.sh\"
+              ],
+              \"environment\": [
```

Comment:
Need to add additional env variable here: `DATABASE_NAME` set to `STAGING_DB`. Without it, the refresher tasks don't correctly check if a matching task is already executing and begin to pile up.